test/index.js

```diff
@@ -28,12 +28,12 @@ const runTest = function(url) {
         error: '>=2000'
       },
       psi: {
-        warn: '>=1500',
-        error: '>=2000'
+        warn: '>=2000',
+        error: '>=3500'
       },
       vc85: {
-        warn: '>=1500',
-        error: '>=2000'
+        warn: '>=3500',
+        error: '>=5000'
```

Comment:
I've never done it myself, but how long does the site-publishing step take? (i.e. does it matter if you accidentally do things in the wrong order, meaning you need to publish the site a second time?)

It's a bit out of date (and incomplete), but @jviotti has a list of the publishing steps he follows [here](https://github.com/resin-io/etcher/blob/master/docs/PUBLISHING.md).